seed/views/v3/users.py

```diff
@@ -192,31 +193,22 @@ def create(self, request):
         # Add the user to the org.  If this is the org's first user,
         # the user becomes the owner/admin automatically.
         # see Organization.add_member()
+
+        try:
+            role = get_role_from_js(role)
+        except Exception:
+            return JsonResponse({
+                'status': 'error', 'message': 'valid arguments for role are [viewer, member, owner]'
+            }, status=status.HTTP_400_BAD_REQUEST)
         if not org.is_member(user):
             try:
-                org.add_member(user, access_level_instance_id)
+                org.add_member(user, access_level_instance_id, role)
             except IntegrityError as e:
                 return JsonResponse({
                     'status': 'error',
                     'message': str(e)
                 }, status=status.HTTP_400_BAD_REQUEST)
 
-        if body.get('role'):
-            # check if this is a dict, if so, grab the value out of 'value'
-            role = body['role']
-            try:
-                get_role_from_js(role)
-            except Exception:
-                return JsonResponse({'status': 'error', 'message': 'valid arguments for role are [viewer, member, '
-                                                                   'owner]'},
-                                    status=status.HTTP_400_BAD_REQUEST)
-
-            OrganizationUser.objects.filter(
-                organization_id=org.pk,
-                user_id=user.pk,
-                access_level_instance_id=access_level_instance_id,
-            ).update(role_level=get_role_from_js(role))
-
```

Comment:
`org.add_member()` has a default `role` value of `ROLE_OWNER`. This caused errors when attempting to create. The role is later applied in an `OrganizationUser` query, however due to the error the code never reached this point